Added multiple project call out (#6727)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/projects.md

After configuring the Project subdirectory option, dbt Cloud will use it as the root directory for your dbt project. This means that dbt commands, such as `dbt run` or `dbt test`, will operate on files within the specified subdirectory. If there is no `dbt_project.yml` file in the Project subdirectory, you will be prompted to initialize the dbt project.

> :::info Multiple projects in dbt Cloud

Reviewer: what do we think about moving this callout higher up? it feels like it's a big buried lower in the page?

maybe after the first paragraph at the top of the page?

> A dbt project informs dbt about the context of your project and how to transform your data (build your data sets). By design, dbt enforces the top-level structure of a dbt project such as the dbt_project.yml file, the models directory, the snapshots directory, and so on. Within the directories of the top-level, you can organize your project in any way that meets the needs of your organization and data pipeline.

> :::info Multiple projects in dbt Cloud
.....text..
:::

> At a minimum, all a project needs is the....

Author: Hiya @mirnawong1 

Thanks for suggesting this. This was one of my initial thoughts but I (again) flittered between the two options lol.

Kind Regards
Natalie

> :::info Multiple projects in dbt Cloud

Reviewer: non blocker and sugg a diff title since the purpose of the callout is to inform users about the project availability based on plans
```suggestion
:::info Project support in dbt Cloud plans
```

> Some [plans](https://www.getdbt.com/pricing) only allow for one project, but you can have multiple projects (and cross-project references with [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-intro)) in Enterprise.

Reviewer: ```suggestion
Some [plans](https://www.getdbt.com/pricing) support only one dbt project, while [Enterprise plans](https://www.getdbt.com/contact) allow multiple projects and [cross-project references](/best-practices/how-we-mesh/mesh-1-intro) with dbt Mesh.
```
